pkg/bmc/redfish.go

```diff
@@ -9,6 +9,7 @@ func init() {
 	registerFactory("redfish", newRedfishAccessDetails)
 	registerFactory("redfish+http", newRedfishAccessDetails)
 	registerFactory("redfish+https", newRedfishAccessDetails)
+	registerFactory("redfish+virtualmedia", newRedfishVirtualMediaAccessDetails)
```

Comment:
What's the logic behind the `+` in `redfish+virtualmedia`? I'd have expected `redfish-virtualmedia` since that's the name of the protocol, and save the `+` for specifying http vs https transport.

Reply:
Super good point. :\

I wonder if we can always assume https back-end transport with virtual media?

Reply:
Addressed in #357

Reply:
> I wonder if we can always assume https back-end transport with virtual media?

I'd like to say yes, but from experience I can report that this conversation always ends with someone whining about CA infrastructure :(